tasks/internal_reconcile_report_mismatch/README.md

```diff
@@ -0,0 +1,50 @@
+[![Known Vulnerabilities](https://snyk.io/test/github/nasa/cumulus-orca/badge.svg?targetFile=tasks/internal_reconcile_report_mismatch/requirements.txt)](https://snyk.io/test/github/nasa/cumulus-orca?targetFile=tasks/internal_reconcile_report_mismatch/requirements.txt)
+
+**Lambda function internal_reconcile_report_mismatch **
+
+Receives job id and page index from end user and returns reporting information of files that are missing from ORCA S3 bucket or have different metadata values than what is expected. 
+
+Visit the [Developer Guide](https://nasa.github.io/cumulus-orca/docs/developer/development-guide/code/contrib-code-intro) for information on environment setup and testing.
+
+- [Deployment](#deployment)
+- [Input/Output Schemas and Examples](#input-output-schemas)
+- [pydoc internal_reconcile_report_mismatch](#pydoc)
+
+<a name="deployment"></a>
+## Deployment
+```
+    see bin/build.sh to build the zip file. Upload the zip file to AWS.
+    
+```
+<a name="input-output-schemas"></a>
+## Input/Output Schemas and Examples
+Fully defined json schemas written in the schema of https://json-schema.org/ can be found in the [schemas folder](schemas).
+
+### Example Input
+```json
+{
+  "jobId": 123,
+  "pageIndex": 0
+}
+```
+### Example Output
+```json
+{
+  "jobId": 123,
+  "anotherPage": false,
+  "mismatches": [
+    {
+      "collectionId": "MOD09GQ___061",
```

Comment:
Missing fields based on current output schema. See discussion items on output as this may change more.